src/Events/Integrations/Plugins/Elementor/Controller.php

```diff
@@ -79,6 +79,11 @@ public function load(): void {
 		tribe( Template_Controller::class );
 
 		$this->register_assets();
+
+		/**
+		 * Fires after the TEC Elementor integration has been loaded.
+		 */
+		do_action( 'tec_events_elementor_loaded' );
```

Comment:
```suggestion
		/**
		 * Fires after the TEC Elementor integration has been loaded.
		 *
		 * @since TBD
		 */
		do_action( 'tec_events_elementor_loaded' );
```